qa/tasks/ceph.py

```diff
@@ -255,10 +255,27 @@ def write_rotate_conf(ctx, daemons):
                 not (ctx.config.get('archive-on-error') and ctx.summary['success']):
             # and logs
             log.info('Compressing logs...')
-            ctx.cluster.sh(
-                'sudo find /var/log/ceph -name *.log -print0 | '
-                'sudo xargs -0 --no-run-if-empty -- gzip --',
-                wait=False)
```

Comment:
sorry for the regression! but how about changing `False` to `True`.

Reply:
That is not going to work.  As mentioned in the commit message, we need parallel execution here.  wait=True would cause us to go serially, waiting for node X logs to be compressed before starting with node Y.

Reply:
will include the suggested change in a follow-up PR.

Reply:
I don't think there is anything to change here.  `cluster.run()` is perfectly fine, `cluster.sh()` is not needed here.